Bylaws.md

```diff
@@ -59,6 +59,9 @@ Categories of Forum Membership are defined as follows:
    * Certificate Consumer: The member organization meets the requirements for membership as a Certificate Consumer in one or more CWGs.
 * Associate Member: As defined in Section 3.1
 * Interested Party: As defined in Section 3.2
+* Probational Certificate Issuer: A Certificate Issuer applying for membership but awaiting full qualification under Section 2.1. During the probational state, the organization has no voting rights. 
```

Comment:
Just going by how I understand the text and in particular section 3.1 before this series of edits, "Participation as Associate Member is by invite only." Has this been handled in some other way in practice?

In any case, by the new text, it is unclear how any entity becomes a "Probational Certificate Issuer". Certainly by withholding a signed IPR upon joining (5.5). But otherwise, is any entity that tries to join any CWG and somehow "awaits full qualification" automatically a "Probational Certificate Issuer" Member of the Forum?

Should we maybe explicitely delegate the question of when and to whom this applies to the CWG Charters? And in addition, should we keep the requirement (for CWG Charters to fulfill) that such status is attainable "by invite only"?

Reply:
> Just going by how I understand the text and in particular section 3.1 before this series of edits, "Participation as Associate Member is by invite only." Has this been handled in some other way in practice?

If you are talking about Associate Members such as WebTrust, ETSI, ACAB'c or FPKI, I don't know, but I assume there was some correspondence years ago. I'm not sure what part of the existing text you find problematic.

> In any case, by the new text, it is unclear how any entity becomes a "Probational Certificate Issuer"

We will follow existing practice, which means that an Applicant that wants to enter the "Certificate Issuer" category, will apply as if everything is met to become a "Certificate Issuer" even if they are missing some of the criteria. They are usually missing the "Trusted by a Certificate Consumer" part. The CWG will evaluate the application and if everything else checks out, the Applicant will become a "Probational Certificate Issuer".

Hope this is a bit clearer.

> Should we maybe explicitely delegate the question of when and to whom this applies to the CWG Charters? And in addition, should we keep the requirement (for CWG Charters to fulfill) that such status is attainable "by invite only"?

We should keep the "invite only" for Associate Members that are more of the "WebTrust", "ETSI" type and not for "CA Issuers". This is what this change is trying to address.

Reply:
I agree with your comment in so far as I agree that what you describe is a reasonable outcome and mirrors the current practice as I understand it. But I think we should maybe try to work on the language to make this even more clear.

Reply:
The application process for membership will be further clarified in the individual WG charters and the template in the Bylaws. We will try to make the expectations for membership applications more clear there.